sdk/examples/client/high_level/consolidation.rs

```diff
@@ -3,41 +3,49 @@
 
 //! In this example we will consolidate all funds in a range of addresses.
 //!
-//! `cargo run --example consolidation --release`
+//! Rename `.env.example` to `.env` first, then run the command:
+//! ```sh
+//! cargo run --release --example address_consolidation [ADDRESS INDEX START] [ADDRESS COUNT]
```

Comment:
Why `address_consolidation`? The outputs get consolidated (merged into a single one)

Reply:
funds are merged as well ... onto a single address, no?

Reply:
Wouldn't say the focus is the latter, you can do it with a single address

Reply:
But don't care that much, in 2.0 I deleted it already anyways :trollface: